src/common/LoggedInProviders.tsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 
 import { ActiveAccountContextProvider } from '../hooks/useActiveAccount';
+import { NotificationsManagerProvider } from '../hooks/useNotificationManager';
+import { UnreadMessagesContextProvider } from '../hooks';
 
 export const LoggedInProviders = ({
   children,
 }: {
   children?: React.ReactNode;
 }) => {
   return (
-    <ActiveAccountContextProvider>{children}</ActiveAccountContextProvider>
+    <NotificationsManagerProvider>
```

Comment:
I struggled to get the useNotification query to actually wait until the app was authenticated on initial launch. The `enabled` flag did not appear to be respected (which probably needs a closer look). Moving these providers here works well and is really where they below.

Reply:
I feel like this is an awesome pattern and we should adopt this for more cases.

Reply:
Yep, agreed